moviebot/controller/http_data_formatter.py

```diff
@@ -0,0 +1,133 @@
+"""This file contains methods to format data for HTTP requests."""
+
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List
+
+from moviebot.agent.agent import DialogueOptions
+
+HTTP_OBJECT_MESSAGE = Dict[str, Dict[str, str]]
+
+SHORT_ANSWER = {
+    "I like this recommendation.": "I like this",
+    "I have already watched it.": "Seen it",
+    "Tell me more about it.": "Tell me more",
+    "Recommend me something else please.": "Something else",
+    "Tell me something about it.": "More information",
+    "/restart": "Restart",
+    "I would like a similar recommendation.": "Similar",
+    "I want to restart for a new movie.": "Restart",
+    "I would like to quit now.": "Quit",
+}
+
+
+def shorten(input: str) -> str:
```

Comment:
Consider making this a private function `_shorten`.

Reply:
Done.